### PR TITLE
Warn users of deprecated classes

### PR DESCRIPTION
**Pull request by gonzaponte**
- ELLookupTable
- RadiusDependentDriftField
- WavelengthShifting

### CODE REVIEW

**Comment by paolafer** [comment]
I'm thinking that we should also remove the addition of the `WavelengthShifting` process from `NexusPhysics`, so there's no doubt that we are not using it anymore. Can you do that?

**Comment by gonzaponte** [comment]
Yes. Why is it that we don't see this process even though it is added to the list?

**Comment by paolafer** [comment]
Because our custom-made process uses properties with different names than those defined in the `TPB` material. Right now there are no optical material properties with the appropriate parameters, so that process never gets to do anything.

File: source/physics/RadiusDependentDriftField.cc [hunk 1]
```diff
@@ -3,6 +3,11 @@
 //
 // Drift field varying with radial coordinate.
 //
+//
```
Comment (paolafer) [inline source/physics/RadiusDependentDriftField.cc]: Maybe, rather than deprecated, this class is still not implemented, right?

**Review by paolafer — APPROVED** [review]
This PR clarifies which classes are not being used in nexus, to avoid confusion for users.

**Pull request merged**